altair/utils/schemapi.py

```diff
@@ -40,16 +40,37 @@ def __init__(self, *args, **kwds):
         object.__setattr__(self, '_args', args)
         object.__setattr__(self, '_kwds', kwds)
 
-    def copy(self, deep=False):
-        """Return a deep copy of the object"""
-        args = self._args
-        kwds = self._kwds
+    def copy(self, deep=True, ignore=()):
```

Comment:
I have read through the code for copy here, and it looks fine, but I would feel better if there were a few simple tests that covered the core cases of SchemaBase, list, dict, with ignored keys.

Reply:
I'll add those – thanks!